.github/workflows/CI.yaml

```diff
@@ -50,3 +51,9 @@ jobs:
         uses: coverallsapp/github-action@master
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
+      
+      - name: Publish 
+        if: ${{ success() && github.event_name == 'push' && github.ref == 'refs/heads/master'}}
```

Comment:
How do you ensure that the other workflow runs first?

Reply:
Which one? We have only one workflow.

Reply:
Sorry, I thought this was an update to the CD workflow. I missed that are only using one now. 
This is a nice solution, good work.